packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -310,17 +310,22 @@ class FlutterPlugin implements Plugin<Project> {
                 localEngineSrcPath this.localEngineSrcPath
                 targetPath target
                 previewDart2 previewDart2Value
-                strongMode strongModeValue	
+                strongMode strongModeValue
                 preferSharedLibrary preferSharedLibraryValue
                 sourceDir project.file(project.flutter.source)
                 intermediateDir project.file("${project.buildDir}/${AndroidProject.FD_INTERMEDIATES}/flutter/${variant.name}")
                 extraFrontEndOptions extraFrontEndOptionsValue
                 extraGenSnapshotOptions extraGenSnapshotOptionsValue
             }
 
+            Task deleteOutputDir = project.tasks.create(name: "deleteOutputDirectory${variant.name.capitalize()}", type:Delete) {
+                delete variant.mergeAssets.outputDir
+            }
+
             Task copyFlxTask = project.tasks.create(name: "copyFlutterAssets${variant.name.capitalize()}", type: Copy) {
                 dependsOn flutterTask
                 dependsOn variant.mergeAssets
+                dependsOn deleteOutputDir
```

Comment:
You should be able to replace this line with
```gradle
            dependsOn "clean${variant.mergeAssets.name.capitalize()}"
```

Reply:
Done